src/main/java/LiUni/KeyHandler.java

```diff
@@ -0,0 +1,69 @@
+package LiUni;
+
+import javax.xml.parsers.DocumentBuilder;
+import javax.xml.parsers.DocumentBuilderFactory;
+import org.w3c.dom.Document;
+import org.w3c.dom.Element;
+import org.w3c.dom.NodeList;
+import org.w3c.dom.Node;
+
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+
+
+public class KeyHandler {
+    public void setupKeys() {
+        DocumentBuilderFactory dbFactory = DocumentBuilderFactory.newInstance();
+        try {
+            File hcKeys = new File("hardcoded_keys.xml");
+            DocumentBuilder dBuilder = dbFactory.newDocumentBuilder();
+            Document doc = dBuilder.parse(hcKeys);
+            doc.getDocumentElement().normalize();
+
+            NodeList serviceList = doc.getElementsByTagName("service");
+            for (int i = 0; i < serviceList.getLength(); i++) {
+                Node service = serviceList.item(i);
+                Element element = (Element) service;
+                String serviceName = element.getAttribute("id");
+
+                //Ideally would use switch case based on what serviceName is
+                if (serviceName.equals("Twitter")) {
+                    System.out.println("Setting up " + serviceName + " service...");
+                    setupTwitter(element);
+                } else {
+                    System.out.println("Unsupported service presented. " + serviceName);
+                }
+            }
+
+        }
+        catch (Exception e) {
+            System.out.println("Could not set up the keys. " + e.toString());
+            e.printStackTrace();
+        }
+    }
+
+    private static void setupTwitter(Element element) {
+        try {
+            BufferedWriter writer = new BufferedWriter(new FileWriter("twitter4j.properties"));
```

Comment:
look into being able to guarantee the closure of the writer in the event an exception occurs within the try/catch block